### ISSUE TITLE
utils.unit.createMocks() throws Exception TypeError: Cannot redefine property: readyHandler on second call

### ISSUE THREAD

**Issue by MiSchroe** [ioBroker/testing#289]
# Issue description

My adapter logic is split into several classes each of them with its own test cases. According to the README I should call `utils.unit.createMocks()` to create my own set of mock objects. That works for the first file with test cases but in the second file I have to create the mock objects again and I receive an exception `TypeError: Cannot redefine property: readyHandler`.
 
# Expected behaviour

Subsequent calls to `utils.unit.createMocks()` should work without throwing an exception.

# Steps to reproduce

1. Generate a new Typescript-based adapter with the template generator CLI.
2. Overwrite the file `main.test.ts` with the code below.
````typescript
/**
 * This is a dummy TypeScript test file using chai and mocha
 *
 * It's automatically excluded from npm and its build output is excluded from both git and npm.
 * It is advised to test all your modules with accompanying *.test.ts-files
 */

import { expect } from "chai";
const { utils } = require("@iobroker/testing");
// import { functionToTest } from "./moduleToTest";

describe("module to test => function to test", () => {
	// initializing logic
	const expected = 5;

	it(`should return ${expected}`, () => {
		const result = 5;
		// assign result a value from functionToTest
		expect(result).to.equal(expected);
		// or using the should() syntax
		result.should.equal(expected);
	});

	it(`shouldn't throw an exception`, () => {
		expect(function () {
			const { database, adapter } = utils.unit.createMocks();
		}).not.to.throw();
	});

	it(`shouldn't throw an exception, neither`, () => {
		expect(function () {
			const { database, adapter } = utils.unit.createMocks();
		}).not.to.throw();
	});
});

// ... more test suites => describe
````
3. Run the test with test:ts script -> The first two tests succeed, the last one fails.

# Analyzed so far

The properties are bound to the function `createAdapterMock`, I'm not sure, if this is the intended behaviour. A simple fix probably would be to set the `configurable` attribute to `true` for the properties in the call to `Object.defineProperties`

**Comment by AlCalzone** [comment]
Fixed by #290

**Comment by AlCalzone** [comment]
Fix released in v2.3.0
